src/realtime.d.ts

```diff
@@ -68,7 +65,65 @@ export interface Entity {
   vehicle?: VehicleUpdate;
   is_deleted: boolean;
 }
-
+interface active_period {
+  start?: Date;
```

Comment:
my only concern is that these aren't really dates, they're numbers. 

If you tell typescript that `start` is a `Date`, it suggests that you can use Date methods like `start.toISOString()` or `start.toLocaleDateString()` which would actually crash if you tried it, beacuse `start` is a number

I'm interested in what you think here - I'm not 100% sure what the correct way of doing it is

Reply:
Oops you are right! Sorry about that I caught the error on one of them but must have missed these